services/catalog/rs_server_catalog/user_catalog.py

```diff
@@ -33,6 +34,7 @@
 from starlette.middleware.base import BaseHTTPMiddleware
 from starlette.responses import JSONResponse
 
+PRESIGNED_URL_EXPIRATION_TIME = 1800  # 30 minutes
```

Comment:
The expiration time interval must be a configuration parameter of RS-Server (30 min by default). So it should be set from the outside, a suggestion would be to use an env var. Later on, I guess it would be added as a parameter in the deploying helm chart.